src/pattern-library/util/jsx-to-string.tsx

```diff
@@ -70,38 +84,54 @@ export function jsxToString(vnode: ComponentChildren): string {
     // (eg. from an index or item ID) so it doesn't make sense to include it either.
     let propStr = Object.entries(vnode.props)
       .map(([name, value]) => {
-        // Don't render props that are for internal pattern-library wrapper
-        // component use only
-        if (name.includes('WrapperProp')) {
-          return '';
-        }
-        if (name === 'children') {
+        if (isExcludedProp(name)) {
           return '';
         }
 
-        // When a boolean property is present, render:
-        // 'booleanPropName' => when true
-        // 'booleanPropName={false}' => when false
-        if (typeof value === 'boolean') {
-          return value ? name : `${name}={false}`;
+        if (isJSXElement(value)) {
+          return `${name}={${jsxToString(value)}}`;
         }
 
-        let valueStr;
-        if (typeof value === 'string') {
-          valueStr = `"${escapeQuotes(value)}"`;
-        } else if (typeof value === 'function' && componentName(value)) {
-          // Handle {import("preact").FunctionComponent<{}>} props
-          valueStr = `{${componentName(value)}}`;
-        } else if (isJSXElement(value)) {
-          valueStr = `{${jsxToString(value)}}`;
-        } else if (value && typeof value === 'object') {
-          // Use the prop name instead of [Object object]; it's more helpful
-          valueStr = `{${name}}`;
-        } else if (value) {
-          // `toString` necessary for Symbols
-          valueStr = `{${value.toString()}}`;
+        let valueStr = '';
+
+        switch (typeof value) {
+          case 'boolean':
+            // When a boolean property is present, render:
+            // 'booleanPropName' => when true
+            // 'booleanPropName={false}' => when false
+            valueStr = value ? name : `${name}={false}`;
+            break;
+          case 'string':
+            valueStr = `${name}="${escapeQuotes(value)}"`;
+            break;
+          case 'bigint':
+          case 'number':
+          case 'undefined':
+            valueStr = `${name}={${value}}`;
+            break;
+          case 'function':
+            // This also handles function components
+            valueStr = componentName(value)
+              ? `${name}={${componentName(value)}}`
+              : `${name}={${value.toString()}}`;
+            break;
+          case 'object':
+            // Use the prop name instead of [Object object]; it's more helpful
+            if (value) {
+              valueStr = `${name}={${name}}`;
+            } else {
+              // null is an object
```

Comment:
I didn't know this! 😂 

![image](https://user-images.githubusercontent.com/2719332/217337819-45bdc084-95c7-444b-8a2f-73f76499561d.png)